syntax = "proto3";

package embedding;

service EmbeddingService {
  rpc GetEmbedding (EmbeddingRequest) returns (EmbeddingResponse) {}
}

message EmbeddingRequest {
  string text = 1;
}

message EmbeddingResponse {
  repeated float embedding = 1;
}


//python -m pip install grpcio
//python -m pip install grpcio-tools
//python -m grpc_tools.protoc -I=. --python_out=. --grpc_python_out=. ./proto/embedding.proto
